booking-up-for-beauty/Pratster95/booking_up_for_beauty.go

```diff
@@ -0,0 +1,49 @@
+package booking
+
+import "time"
+import "fmt"
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	myDate, _ := time.Parse("1/2/2006 15:04:05", date)
+	return myDate
+	panic("Please implement the Schedule function")
+}
+
+// HasPassed returns whether a date has passed
+func HasPassed(date string) bool {
+	today := time.Now()
+	//myDateString := "July 25, 2019 13:45:00"
+	myDate, _ := time.Parse("January 2, 2006 15:04:05", date)
+	return today.After(myDate)
+	panic("Please implement the HasPassed function")
+}
+
+// IsAfternoonAppointment returns whether a time is in the afternoon
+func IsAfternoonAppointment(date string) bool {
+	myDate, _ := time.Parse("Monday, January 2, 2006 15:04:05", date)
+	if myDate.Hour() >= 12 && myDate.Hour() < 18 {
+		return true
+	}
+	if myDate.Hour() == 18 && myDate.Minute() == 00 {
+		return true
```

Comment:
I think if the time is 18:00 the method should return false since the question defines afternoon as >= 12:00 and < 18:00